index.js

```diff
@@ -227,6 +249,8 @@ DNSDiscovery.prototype._onquestion = function (query, port, host, answers, multi
   var domain = parseDomain(query.name)
 
   if (query.type === 'TXT' && domain === query.name) {
+    // TODO wait, what is this? is it an unsubscribe? that's the only codepath in _onanswer this seems to match
+    // debug('Replying announce via TXT to', host + ':' + port)
```

Comment:
This is a debugging thing. You can use tools like `dig` to query for a text record the get information about a domain. Useful to sanity check things in production.  Could you remove the comment or update it?